dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -2610,7 +2610,6 @@ private void extendWithAssignmentForConditionalExpr(
         handleArtificialTree(condExprVarUseTree);
         LocalVariableNode condExprVarUseNode = new LocalVariableNode(condExprVarUseTree);
         condExprVarUseNode.setInSource(false);
-        extendWithNode(condExprVarUseNode);
```

Comment:
Should also change the method name and doc?

Reply:
Yes, please. The test is failing, so maybe this is needed somewhere?